### PR TITLE
fix: phpcs errors on initial commits

### PR DESCRIPTION
**Pull request by devtaher**
Bug Fix for  #16

### CODE REVIEW

File: axltempl/files/drupal/load.environment.php [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?php
 
+// @codingStandardsIgnoreFile
 /**
  * This file is included very early. See autoload.files in composer.json and
```
Comment (hussainweb) [inline axltempl/files/drupal/load.environment.php:5]:
```suggestion
/**
 * @file
 * This file is included very early. See autoload.files in composer.json and
```

How about this instead?

File: axltempl/files/drupal/settings.env.php [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?php
 
+// @codingStandardsIgnoreFile
```
Comment (hussainweb) [inline axltempl/files/drupal/settings.env.php:3]:
```suggestion
/**
 * @file
 * Load settings from environment variables.
 * @see /.env.example
 */
```

How about this instead?

**Comment by devtaher** [comment]
@hussainweb Committed changes as per your suggestions.

**Comment by hussainweb** [comment]
Thanks, @devtaher. I take it that `phpcs` doesn't report issues with the updated comments, right?

**Comment by devtaher** [comment]
@hussainweb Yes, I have tested this on local and it does not throw any phpcs errors on commit.

**Comment by hussainweb** [comment]
Thanks for confirming, @devtaher. Merged.

**Pull request merged**